### DISCUSSION TITLE
Balance not updated in Neuron

### DISCUSSION THREAD

**Discussion by hevy75** [nervosnetwork/neuron#2891] [General]
Hello.
I'm sorry i should have missed something somewhere...
I've downloaded and installed Neuron yesterday, Sync goes fine with a 100% completion.
I have send my CKB from Binance to my wallet address
So far so good...
But now the wallet overview is not updating in neuron (balance remain at 0ckb) and i could not even find my wallet or transaction in the history/search area of neuron app even if i could find it in CKB explorer... (i could not find anything in this section even with just a date it remains empty)
Is there something i should do more to get Neuron updated with my lovely CKB ;-)

Thanks a lot !

![Neruron issue](https://github.com/nervosnetwork/neuron/assets/148544946/9c6a7ab8-19ea-4b1f-87a3-23296c3c2d4d)

**Comment by Keith-CY** [comment]
Hi @hevy75 Could you upload debug information of Neuron so we can have an inspection about what happened inside Neuron? The debug information can be exported by **Menu => Help => Export debug information**, and it can be uploaded by drag&drop into the text field.

**Comment by hevy75** [comment]
Hi Keith. 
Of course here it is
[neuron_debug_1697816000665.zip](https://github.com/nervosnetwork/neuron/files/13055915/neuron_debug_1697816000665.zip)

**Reply by Keith-CY** [reply]
First of all, I'm sure you asset is safe and will be revealed in Neuron.

---
You asset was received at block `11,234,458`, so I tracked the debug information around that time and found things unusuall

```
[2023-10-20 10:09:06.614] [info]  Database:	saved synced block #11233214
[2023-10-20 10:09:11.637] [info]  Database:	saved synced block #11237866
[2023-10-20 10:10:37.052] [info]  Database:	saved synced block #11237876
[2023-10-20 10:14:02.923] [info]  Database:	saved synced block #11237886
```

It took 5 seconds to sync block from `11233214` to `11237866`, `4652` blocks in total, including the target block `11,234,458`. It's too fast(compared with other logs). So I think these blocks are lost in the synchronization.

It should be fixed by **Settings => Data => Clear Cache(without `fully rebuild index`)**. This action won't take much time.

**Reply by Keith-CY** [reply]
It's the first time we found that blocks may be lost in synchornization, @yanguoyu please have a look.

**Reply by Keith-CY** [reply]
May I have the entire logs of the built-in CKB node to fully analyze this case? It would be helpful for us to improve the application.

The entire logs can be found by the following steps:
1. Navigate to **Settings => Data**
2. Click the **CKB Node Config & Storage** path, it will open the file explorer
3. Navigate to **data** folder in the file explorer, and a folder named **logs** should be there
4. Compress the folder and send it to neuron@magickbase.com

Thank you

**Reply by hevy75** [reply]
Dear Keith
First of all, Thanks for this swift answer.
I've just started the clear cache and fully rebuild process.
I keep you posted once everything done.

**Reply by Keith-CY** [reply]
My bad, I should highlight the **\*without\*** in the comment :sorry:
Rebuilding the index requires a resync of the built-in ckb node indexer, so it takes a bit longer than expected.

**Reply by hevy75** [reply]
Yes for sure.. i red it too fast, My bad.
Sorry about that

**Reply by hevy75** [reply]
Keith
Re-sync done, but still 0 ckb in balance so i've cleared cache (just refresh this time ;)) and i get my CKB balanced.
Youpi ! 
About logs i have zipped it but get a 1.5Go file so not sure it will be send by mail. 
I upload it on a WeTransfert and will send you the link as soon as it's done.

A huge Thanks for the support (and happy to be on the project)

**Reply by Keith-CY** [reply]
Glad to hear that!

WeTransfer is good, and we are using it, too. You may share the link via e-mail so it won't be public on GitHub if you mind that.

**Reply by hevy75** [reply]
Mail sent. 
Have a great day/evening

**Reply by Keith-CY** [reply]
Roger that. Really appreciate your support and have a good one :)

**Comment by hevy75** [comment]
> May I have the entire logs of the built-in CKB node to fully analyze this case? It would be helpful for us to improve the application.
> 
> The entire logs can be found by the following steps:
> 
>     1. Navigate to **Settings => Data**
> 
>     2. Click the **CKB Node Config & Storage** path, it will open the file explorer
> 
>     3. Navigate to **data** folder in the file explorer, and a folder named **logs** should be there
> 
>     4. Compress the folder and send it to [neuron@magickbase.com](mailto:neuron@magickbase.com)
> 
> 
> Thank you



> 4\. Compress the folder
So far i can't compress, maybe i've to wait until sync is over.

![image](https://github.com/nervosnetwork/neuron/assets/148544946/41837420-1146-41d1-a49e-15180613fcc7)

in French but it saying :
"D:\Actif\CKB\Node\Neuron\data\logs\run.log
The process cannot access the file because it is being used by another process."

**Reply by Keith-CY** [reply]
That's fine, the `run.log` is processed by the built-in ckb node because of the re-sync

**Comment by yanguoyu** [comment]
Hi, @hevy75  do you try to clean cache with the default option?  For Windows, Click `Help -> Settings -> Data -> Clear Cache`, For Mac OS, click `Neuron->Preferences->Data -> Clear Cache`.

![image](https://github.com/nervosnetwork/neuron/assets/12881040/80697c42-0d4b-4176-956f-7f981f240b11)

**Reply by hevy75** [reply]
Dear yanguoyu.
I'm not at home currently but as mentioned that what I have done after Keith advice and it works fine now. I've got my ckb balance correct and they are stacked now.
Do you need to get other files/folder from my side to understand the sync issue I get ?
Happy to help if needed.
Great thanks to both for support.

**Reply by Keith-CY** [reply]
Hi @yanguoyu 
This case has been resolved by `clear cache(without rebuilding index)`, but I want to figure out the reason why blocks were lost during synchronization, so I've opened another issue(https://github.com/Magickbase/neuron-public-issues/issues/306) for analysis, we may have technical discussion in that issue

**Comment by jadicc987** [comment]
Here's a concise and helpful comment for the issue:

---

Hi @hevy75,  

It seems like Neuron hasn't synced correctly or fully recognized your transaction. Here are a few steps to try:  
1. Ensure Neuron is running the latest version to avoid compatibility issues.  
2. Check that the synchronization process has reached 100% and the node is fully synced.  
3. Verify your wallet address in the app matches the one you sent funds to.  
4. If the issue persists, try restarting Neuron or clearing the app's cache.  

Feel free to reach out if you need further assistance. You might also find insights by consulting an *[infection control expert](https://jadicc.com/customized-infection-control-plans/)* to ensure a clean app environment. 😉  

Good luck!

**Comment by hevy75** [comment]
Hello.
Thank you for your email.
I am just wondering if this answer is related to the issue raised in
October 2023 or is it something else ?

My last operation was destacking and apparently everything goes well so if
there is an issue I did not get can you just tell me so I will follow your
instruction as, for the moment, It seems I don't have any problem with
neuron.

Thanks for your support.
